### ISSUE TITLE
No WebTerminal app after updating to 2.1.5

### ISSUE THREAD

**Issue by MrPeacockNLB** [coder/coder#9553]
# Description

Just updated to the latest coder version v2.1.5. **WebTerminal** and **SSH** buttons has gone. Reading #9100 this should be configurable through the terraform provider.

I've also updated to latest release of the provider to configure my apps as this:

```
  display_apps {
    vscode          = true
    vscode_insiders = false
    web_terminal    = true
    ssh_helper      = false
  }
```

This setting should show the WebTerminal but nothing happens. All I can see is Code-Server button and my own apps I've defnined.

![image](https://github.com/coder/coder/assets/103434584/fd57eb31-64cc-46ee-84d2-ed9f10fc1929)

> Note: A running "old" workspace has the WebTerminal and it is still functioning

**Comment by MrPeacockNLB** [comment]
Yes, we are using external provisioner.  Thanks for the hint. We've deployt v0.27.1. It seems we've missed this in our helm charts for provisionerd. I'll update this also to the latest version and let you know if it works again.

**Comment by ericpaulsen** [comment]
@MrPeacockNLB updating should resolve the issue. i just ran into this myself a couple of hours ago. provisioners running versions >2.1.5 are not aware of the `display_apps` code, and thus aren't able to pick-up the `display_apps` configuration.

**Comment by MrPeacockNLB** [comment]
Thanks!

![image](https://github.com/coder/coder/assets/103434584/c3ec8b56-4031-470e-bb0a-24f621df9b3d)

closing as this is fixed by updating provisionerd to v2.1.5

**Comment by bpmct** [comment]
FWIW we plan on adding better alerting when a provisioner is out of date. This is definitely an odd behavior!
